src/app/models/field/emailField.ts

```diff
@@ -1,10 +1,12 @@
-import { isEmpty } from 'lodash'
 import { Schema } from 'mongoose'
 
+import { validateEmailDomains } from '../../../shared/util/email-domain-validation'
 import { IEmailFieldSchema, ResponseMode } from '../../../types'
 
 const createEmailFieldSchema = () => {
-  const EmailFieldSchema = new Schema<IEmailFieldSchema>({
+  const EmailFieldSchema: Schema<IEmailFieldSchema> = new Schema<
+    IEmailFieldSchema
+  >({
```

Comment:
Can add the typing here instead of in the variable to remove the "Missing return type on function" ESLint warning, as that is the canonical way to add type to a function:

```ts
const createEmailFieldSchema = (): Schema<IEmailFieldSchema> => {
  const EmailFieldSchema = new Schema<IEmailFieldSchema>({
  ...
}
```
```

Reply:
Ok, on it!